wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/LicenseService.java

```diff
@@ -174,20 +187,18 @@ public Response isCodenvyUsageLegal() throws ApiException {
         int actualServers = 0;
 
         try {
-            actualUsers = delegate.getNumberOfUsers();
-
-            List<String> additionalNodes = delegate.getNodes().get(Config.SWARM_NODES);
-            if (additionalNodes != null) {
-                actualServers = additionalNodes.size();
-            }
-
+            actualUsers = adminUserDao.getAll(30, 0).getTotalItemsCount();
```

Comment:
It seems there should be another method to get user number, because parameter **30** looks strange here.